app/controllers/seller_gift_cards_controller.rb

```diff
@@ -12,7 +12,9 @@ def show
               :name,
               :email,
               :created_at,
-              :expiration
+              :expiration,
+            ).where(
+              single_use: false
```

Comment:
I think we want to allow for the frontend to specify whether or not it wants gift_a_meal cards or not. Like a query parameter. @Yong-L has an example of ordering for our Sellers — so something akin to that except a filter instead of an order

Reply:
gotcha! can make the change

Reply:
@Yong-L I was wondering if creating a validation file was necessary, I tried it out by adding an if statement to check for parameters to append the .where clause to the query, which worked when I tested it out in postman, but I was wondering if I was missing something by forgoing a separate query generator in the validate folder.